game/missiongenerator/logisticsgenerator.py

```diff
@@ -56,8 +56,9 @@ def generate_logistics(self) -> LogisticsInfo:
             if (
                 waypoint.waypoint_type
                 not in [
-                    FlightWaypointType.PICKUP,
-                    FlightWaypointType.DROP_OFF,
+                    FlightWaypointType.PICKUP_ZONE,
+                    FlightWaypointType.DROPOFF_ZONE,
+                    FlightWaypointType.AIR_DROP,
```

Comment:
Unused

Reply:
Why? Without this the whole Logic of the CTLD implementation does not work

Reply:
This waypoint type is never created:

```powershell
(venv) ~\src\dcs_liberation [waypoint-cleanup ≡]> git grep AIR_DROP
game/ato/flightwaypointtype.py:    AIR_DROP = 31  # Waypoint to drop paylod from the air
game/missiongenerator/aircraft/waypoints/waypointgenerator.py:            FlightWaypointType.AIR_DROP: AirDropBuilder,
game/missiongenerator/logisticsgenerator.py:                    FlightWaypointType.AIR_DROP,
```

There's the definition, and two conditionals checking if a waypoint is an `AIR_DROP`. This waypoint type is never created though, so nothing `AirDropBuilder` does will ever run.

Reply:
Ah it was just about airdrop.. GitHub marked the 3 lines above as well. Airdrop can be removed yes

Reply:
Ah yes, just airdrop.